app/models/contact_email.rb

```diff
@@ -0,0 +1,5 @@
+class ContactEmail < ContactPoint
+  def self.label_collection
```

Comment:
I don't remember the rails convention for the naming of this stuff, but yeah your right. Maybe you can find out. I think there is a 'for' in the name, but i don't remember